platform/view/services/storage/driver/sql/query/pagination/keyset.go

```diff
@@ -50,6 +52,32 @@ func KeysetWithId[I comparable, V id[I]](offset int, pageSize int, sqlIdName com
 	return Keyset[I, V](offset, pageSize, sqlIdName, func(v V) I { return v.Id() })
 }
 
+func (k *keyset[I, any]) Serialize() ([]byte, error) {
+	ret, err := json.Marshal(k)
```

Comment:
I would serialize into byte64 tbh but its a matter of taste

Reply:
what's byte64? What do you mean?